src/cloudinary-selector.js

```diff
@@ -123,18 +97,34 @@ function initCustomElement() {
         {
           cloud_name: config.cloudName,
           api_key: config.apiKey,
-          default_transformations: config.defaultTransformations,
-          button_class: "btn btn--primary",
-          button_caption: "Select Assets",
+          button_class: "btn btn--tertiary",
+          button_caption: "pick from assets",
+          multiple: true,
+          integration: {
+            type: "kontentai_connector",
+            platform: "kontent_custom_element 1.0",
+            version: "1.0",
+            environment: "prod"
+          }            
         },
         {
           insertHandler: function (data) {
             updateValue(data.assets);
           },
+          showHandler: function() {
+            updateSize(800);
+          },
+          hideHandler: function() {
+            updateSize();
+          }
         },
-        document.getElementById("open-btn")
+        "#open-btn"
       );
 
+      if(config.defaultTransformations) {
+        window.ml.default_transformations = config.defaultTransformations;
+      }
+
```

Comment:
setting the default_transformations property won't have any effect here, as the media library is already instantiated above. as a result, transformations are not set
```suggestion
```